src/pal/prebuilt/inc/corerror.h

```diff
@@ -27,15 +27,10 @@
 #define CLDB_S_NULL SMAKEHR(0x1109)
 #define CLDB_S_INDEX_TABLESCANREQUIRED SMAKEHR(0x1125)
 #define TLBX_I_TYPEINFO_IMPORTED SMAKEHR(0x116c)
-#define TLBX_I_PIA_REGISTERED_FOR_TLB SMAKEHR(0x116d)
-#define TLBX_I_AGNOSTIC_ASSEMBLY SMAKEHR(0x116e)
 #define TLBX_I_USEIUNKNOWN SMAKEHR(0x116f)
```

Comment:
There is a bunch like `TLBX_I_USEIUNKNOWN` that is de-facto unused as well. 

These are only used in debug-only code in `ex.cpp` that is trying to format `HRESULT` as string. If the only use is in this debug-only code, it is fine to delete it as well.